Test/BSN.Commons.AutoMapper.Tests/CommonMapperProfileTests .cs

```diff
@@ -0,0 +1,51 @@
+﻿using AutoMapper;
+using BSN.Commons.AutoMapper;
+using BSN.Commons.Responses;
+
+namespace BSN.Commons.Tests
+{
+    [TestFixture]
+    public class CommonMapperProfileTests
```

Comment:
Please add one another test, that shows working this AutoMapper helper with additional custom AutoMapper profile